tests/cmake_integration/CMakeLists.txt

```
@@ -9,3 +9,6 @@ add_subdirectory(Nested_IDL)
 if(NOT OPENDDS_STATIC)
   add_subdirectory(generated_global)
 endif()
+if(OPENDDS_NO_OPENDDS_SAFETY_PROFILE EQUAL NOTFOUND OR OPENDDS_NO_OPENDDS_SAFETY_PROFILE EQUAL ON)
```

`NOTFOUND` and `ON` are both boolean literals (out of many) in CMake. `NOTFOUND` is false and `ON` is true, so I'm a little confused by what you intended, but going the test file, I'm assuming:

```suggestion
if(OPENDDS_NO_OPENDDS_SAFETY_PROFILE)
```